service-runner/build.gradle.kts

```diff
@@ -10,6 +10,7 @@ plugins {
 }
 
 dependencies {
+  implementation(libs.snakeyaml)
```

Comment:
❓ Do we use `snakeyaml`?

Reply:
It is a dependency from springboot but the version used by springboot has a vulnerability. I'm manually adding it to make sure we can control the version of this dependency and force springboot to use our version.

Reply:
Thanks for the explanation. 

Does it make sense to upgrade the springboot dependency version?

Reply:
Can you also leave a comments in the snake yaml dependency? 

If you think it is too much to upgrade spring boot, it is also very ok with me. 

Please feel free to merge when you think it is ok.

Reply:
There was no available version of springboot where they update that vulnerability themselves, that's why I had to force the update.

I'll check if they've released a new update and if not, I'll add comments as you suggested 👍